04 Abstraction Exercises Part 1/PublicVehicle/src/main/java/com/project/martinez/PublicVehicle.java

```diff
@@ -0,0 +1,30 @@
+package com.project.martinez;
+
+public class PublicVehicle {
+    private double distance;
+    private double fare;
```

Comment:
why is fare still here?

Reply:
nag eerror po yung setFare at getFare pa inaalis po